src/doc/dev_manual/LiveCustomerResponse.rst

```diff
@@ -178,12 +195,16 @@ the associated GitHub issue in response to the *initial* email, the less likely
 this can occur. In addition, boilerplate guidance in an issue template will help
 to mitigate this by informing ``visit-users`` members of this behavior.
 
-For any work the **Primary** engages in, there should be an associated issue
-for tracking that work.
+For any work the **Primary** performs, even if it is a rather trivial amount
+of work to resolve, there should be an associated issue for tracking that work.
+Tracking even the trivial issues can help identify patterns that we may be able
+to later mine to identify process improvements.
 
 .. danger::
    What about ``visit-help-asc`` email list? Should we handle similarly?
```

Comment:
Related to my comment above about the visit-support github account, we could create another account for this email too: visit-support-1 and visit-support-2, etc. Alternatively, since we want to move away from email support in the long run, a first step here could be to consolidate the different email addresses into one: visit-users. This will not be as big of a change to the users since it just requires using a different email address and not needing to create a github account.

Reply:
> a first step here could be to consolidate the different email addresses into one: visit-users.

I think we need to have a broader team discussion about this. I will add an issue ticket